### ISSUE TITLE
why is the website offline?

### ISSUE THREAD

**Issue by davidak** [crateio/crate.io#18]
https://crate.io/packages/

it looked very clean and professional:

http://archive.is/D4doB
http://archive.is/WgQrx

this still works: https://pypi.crate.io/simple/Flask/

**Comment by esc** [comment]
Yes, crate.io now seems to host a big data company, what happend?

**Comment by xiongchiamiov** [comment]
It appears to have sorta moved to https://preview-pypi.python.org/ , except missing features and with more 500s. :(

**Comment by davidak** [comment]
That is hosted by OSUOSOL with lots of other Open Source-Projects.
http://osuosl.org/communities
![bildschirmfoto 2014-01-25 um 00 14 18](https://f.cloud.github.com/assets/91113/1999936/6afcd8a8-854d-11e3-9624-f001e6bcd0e8.png)
Very interesting, i don't knew that.

**Comment by chhantyal** [comment]
Can anyone answer and clarify this issue? @dstufft

**Comment by r1chardj0n3s** [comment]
Donald and I are working on Warehouse, which is what you see at https://preview-pypi.python.org/ and is his new effort supplanting crate.io.

**Comment by davidak** [comment]
@r1chardj0n3s so it would be nice to have a link to the Warehouse repo in the readme of this and a short description about the status.

**Comment by dstufft** [comment]
Copying my answer from reddit:

Basically a combination of things.

I was not putting any time and effort into it as I was now working on PyPI itself. Meanwhile it was broken, I was getting several error emails a day that required deep architectural changes to fix. I've been on the fence about leaving it up in a degraded state or finally killing it off so that people weren't using it expecting that I was maintaining it. On top of that it got maybe 100 hits a day (compared to the ~90k that PyPI gets typically).

The clincher that got me to finally get around to killing it off officially was when a company offered to buy the domain from me.

So in the future we'll end up with the improvements that Crate had on PyPI itself, where a far greater number of people can benefit from them.
